furnace.c

```diff
@@ -501,10 +537,15 @@ main_work_loop(void)
   init_pilot(ctx);
   init_stdio(ctx);
 
+#if MAGNETRON_ON
+  init_magnetron(ctx);
+#endif
+
   cyw43_arch_gpio_put(CYW43_WL_GPIO_LED_PIN, 1);
 
   while (1) {
-    const bool deadline_met = get_absolute_time() > ctx->update_deadline;
+    absolute_time_t abs = get_absolute_time();
```

Comment:
Consider renaming `abs` => `now`. It's much more common to call it like that. We might get some clash with `abs()` from libc.